pkg/otlp/model/translator/histograms_test.go

```diff
@@ -0,0 +1,74 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022-present Datadog, Inc.
+
+package translator
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+)
+
+func TestDeltaHistogramOptions(t *testing.T) {
+	tests := []struct {
+		name     string
+		otlpfile string
+		ddogfile string
+		options  []Option
+	}{
+		{
+			name:     "simple histogram distributions",
```

Comment:
Can we follow [the guidelines](https://datadoghq.atlassian.net/wiki/spaces/OTEL/pages/2580119947/Go+Coding+Guidelines#Tests) for test names? Alternatively, it also works to not have a name and use `""` as the first argument to `t.Run("", ...`, which causes the runner to index the tests and you can still come back to the potentially failing one by counting.

Reply:
Tried to make these shorter